tests/functional/export_v2.t

```diff
@@ -65,3 +65,25 @@ Export with auspice config JSON with an extensions block
   $ python3 "$TESTDIR/../../scripts/diff_jsons.py"  export_v2/dataset2.json "$TMP/dataset3.json" \
   >   --exclude-paths "root['meta']['updated']"
   {}
+
+# export_v2/location_node-data2.json was produced by a mismatched version
+# Running without --skip-validation should result in an error
+# Message printed: "Validation of auspice_config3.json failed."
+  $ ${AUGUR} export v2 \
+  >   --tree export_v2/tree.nwk \
+  >   --node-data export_v2/div_node-data.json export_v2/location_node-data2.json \
+  >   --auspice-config export_v2/auspice_config3.json \
+  >   --output "$TMP/dataset4.json" &>/dev/null
+  [1]
+
+# Skipping validation gives us the same results as `location_node-data.json`
+  $ ${AUGUR} export v2 \
+  >   --tree export_v2/tree.nwk \
+  >   --node-data export_v2/div_node-data.json export_v2/location_node-data2.json \
+  >   --auspice-config export_v2/auspice_config2.json \
+  >   --output "$TMP/dataset5.json" \
+  >   --skip-validation &>/dev/null
```

Comment:
Same as above comment, I would not pipe stderr to null. If `--skip-validation` works, we shouldn't get any error messages and the exit code should be 0.

Reply:
Addressed in [404595d](https://github.com/nextstrain/augur/pull/902/commits/404595d60759a46f3294154ec37211d5ea907feb) -- the output is now detailed in all it's beauty 😂